packages/cli-kit/src/public/node/ruby.ts

```diff
@@ -41,7 +41,9 @@ interface ExecCLI2Options {
  * @param options - Options to customize the execution of cli2.
  */
 export async function execCLI2(args: string[], options: ExecCLI2Options = {}): Promise<void> {
-  const embedded = (await isShopify()) || isTruthy(process.env.SHOPIFY_CLI_EMBEDDED_THEME_CLI)
+  const embedded =
+    ((await isShopify()) && process.env.SHOPIFY_CLI_EMBEDDED_THEME_CLI === undefined) ||
```

Comment:
What happens if I set `SHOPIFY_CLI_EMBEDDED_THEME_CLI=false` or `SHOPIFY_CLI_EMBEDDED_THEME_CLI=0`? I'd recommend adding an `isFalsy` utility like [this one](https://github.com/Shopify/cli/blob/ad5ba3eb7260815cd8561c96770677719b16f178/packages/cli-kit/src/private/node/context/utilities.ts#L4) for truthiness and using that one instead.

Reply:
Also, it's a bit unclear what `embedded` means here and why the logic changed from what we had before. What's the connection between "embedded" and (is Shopifolk and SHOPIFY_CLI_EMBEDDED_THEME_CLI is not defined)?